7985

```diff
@@ -0,0 +1,13 @@
+https://github.com/refined-github/refined-github/pull/7985
+
+```js
+JS:
+issues: // #12, GH-34, fregante#1 and facebook/react#5678
+url: // https://github.com/?q=refined-github
+```
```

Comment:
# Review comment

## Code blocks


Native links:

- https://github.com
- https://google.com/?q=never+gonna

```js
JS:
issues: // #12, GH-34, fregante#1 and facebook/react#5678
url: // https://github.com/?q=refined-github
```

```
No syntax:
issues: // #12, GH-34, fregante#1 and facebook/react#5678
url: // https://github.com/?q=refined-github
```

## Code blocks in table

<table><tr><td>


Native links:

- https://github.com
- https://google.com/?q=never+gonna


```js
JS:
issues: // #12, GH-34, fregante#1 and facebook/react#5678
url: // https://github.com/?q=refined-github
```

```
No syntax:
issues: // #12, GH-34, fregante#1 and facebook/react#5678
url: // https://github.com/?q=refined-github
```

</table>

## Inline code

https://github.com/refined-github/sandbox/blob/7becc26eccef5da79f64003356067e089043d8fb/README.md#L17-L24